src/components/journey/errors/JourneyErrorFallback.tsx

```
@@ -0,0 +1,48 @@
+          <p className="text-white/80 mb-6 max-w-md">
+            Something went wrong with the journey planner. Please try refreshing
+            the page.
+          </p>
+          <button
+            onClick={handleRefresh}
+            className="bg-white text-red-700 px-6 py-2.5 rounded-md 
+            hover:shadow-lg transition-all duration-300 flex items-center gap-2 font-medium"
+            type="button"
+            aria-label="Refresh page"
+          >
+            <i className="fas fa-redo-alt" aria-hidden="true"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)